bodhi-messages/tests/test_update.py

```diff
@@ -635,6 +644,8 @@ def test_requirements_met_stable_v1(self):
                 },
             }
         )
+        print(repr(str(msg)))
```

Comment:
I think these `print` aren't part of the change?

Reply:
Ah crap I thought I got rid of them. Thanks!